WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/SignupFlow.java

```diff
@@ -87,5 +88,6 @@ public SignupFlow dismissInterstitial() {
     public void confirmSignup() {
         // Confirm signup
         waitForElementToBeDisplayed(R.id.nav_sites);
+        new MySitesPage().go();
     }
```

Comment:
Before this commit, the test was passing successfully. Then activity was finishing while recylerview is still refreshing and then [ReaderExpandableTagsView](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderExpandableTagsView.kt) on "Reader" screen was crashing because of unaccessible Hilt objects.

If test was waiting enough for reloading of recyclerview, test was passing. But sometimes it was failing.

Waiting for 2 seconds at `confirmSignup()` can prevent failing.
I chose navigating "My Sites" screen that also prevent failing.

Reply:
Just replicating here what I said on Slack, for visibility: I personally think both approaches are fine in this case, but maybe we should add an extra comment there, because it’s not immediately obvious that removing that piece of code could cause the test to fail.